cmd/minishift/cmd/config/config.go

```diff
@@ -123,6 +123,10 @@ var (
 	SkipCheckOpenShiftRelease = createConfigSetting("skip-check-openshift-release", SetBool, nil, nil, true, nil)
 	WarnCheckOpenShiftRelease = createConfigSetting("warn-check-openshift-release", SetBool, nil, nil, true, false)
 
+	// Pre-flight checks for artifacts (before start)
```

Comment:
Artifacts is too general term here. It should be 
```
// Pre-flight checks for oc (before start)
```

Reply:
good enough...

pre-flight checks refactor is coming anyway...

Reply:
The general term is intentional here as the section is for artifacts and this point we have only for oc but in future we might have more checks e.g. for ISO

Reply:
@budhrg it should be like that atm we only testing for `oc` but in future we also have some test for `iso`  and all those artifacts tests can be part of this group.

Reply:
IMO, we can always refactor later but the current comment is too generalized. It might happen that pre-flight check for ISO may take time. It's always good to represent present. That's my opinion. 

This review comment shouldn't block PR though.